components/DocumentModels.vue

```diff
@@ -0,0 +1,149 @@
+<script setup lang="ts">
+import type { IAllDocumentModels, TrackModel } from "@bcc-code/bmm-sdk-fetch";
+import type { IDiscoverableGroup } from "~/composables/discover";
```

Comment:
This type should be moved in here since it's not used in the composable anymore.

Reply:
done